presentation/src/main/java/com/nexters/knownknowns/presentation/ui/theme/Color.kt

```diff
@@ -1,11 +1,315 @@
 package com.nexters.knownknowns.presentation.ui.theme
 
+import androidx.compose.runtime.Stable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.setValue
 import androidx.compose.ui.graphics.Color
 
-val Purple80 = Color(0xFFD0BCFF)
-val PurpleGrey80 = Color(0xFFCCC2DC)
-val Pink80 = Color(0xFFEFB8C8)
+// Background
+val BackgroundBase = Color(0xFFF7F8FA)
+val BackgroundSurface = Color(0xFFFFFFFF)
+val BackgroundElevated = Color(0xFFFFFFFF)
+val BackgroundOnSurface = Color(0xFFF7F8FA)
+val BackgroundOnElevated = Color(0xFFF7F8FA)
+val BackgroundSurfaceInverse = Color(0xFF1B1D1F)
+val BackgroundDimmed = Color(0xB3000000)
```

Comment:
시멘틱 컬러에 사용하는 Color 객체는 하드코딩 대신 피그마와 동일하게 컬러 팔레트를 먼저 구성한 뒤 사용하면 좋을 것 같아! 아래는 예시 코드!
```suggestion
// 팔레트
val White = Color(0xFFFFFFFF)
val Gray10 = Color(0xFFFAFBFC)
val Gray30 = Color(0xFFF7F8FA)
val Gray50 = Color(0xFFF2F4F7)
val Gray100 = Color(0xFFD7DCE0)
// ...
val Gray900 = Color(0xFF1B1D1F)
val Black = Color(0xFF000000)

// Background
val BackgroundBase = Gray30
val BackgroundSurface = White
val BackgroundElevated = White
val BackgroundOnSurface = Gray30
val BackgroundOnElevated = Gray30
val BackgroundSurfaceInverse = Gray900
val BackgroundDimmed = Black.copy(alpha = 0.7F)
```

Reply:
오 왜 이 방법을 생각 못했을까..!
이렇게 하는 게 확실히 좋은 것 같네용 반영하겠습니다~